### DISCUSSION TITLE
Message Template Buttons

### DISCUSSION THREAD

**Discussion by nickboites** [netflie/laravel-notification-whatsapp#7] [Q&A]
Hi, everyone. I'm trying to send a template message with buttons, but I'm a little bit lost about how to do it. Can you help me?

**Comment by brunopmaia** [comment]
> Hi, everyone. I'm trying to send a template message with buttons, but I'm a little bit lost about how to do it. Can you help me?

I am also in need of this functionality. I analyzed the code and saw that it is not available.

**Comment by beatrizsanchez** [comment]
I've submitted #9 to support buttons. Any feedback is appreciated.
